hack/run-e2e-test.sh

```diff
@@ -73,6 +73,7 @@ then
     OPTS+=(-p)
     ginkgo -mod=mod "${OPTS[@]}" --focus="csi-block-vanilla-parallelized" tests/e2e
 else
+    echo "ginkgo -mod=mod ${OPTS[@]} --focus="$FOCUS" tests/e2e"
```

Comment:
you can remove double quotes around `$FOCUS`

Reply:
done